stable/minio/templates/statefulset.yaml

```diff
@@ -4,7 +4,7 @@
 {{ $drivesPerNode := .Values.drivesPerNode | int }}
 {{ $scheme := "http" }}
 {{- if .Values.tls.enabled }}
-{{ $scheme := "https" }}
+{{ $scheme = "https" }}
```

Comment:
I got error on this change with helm v2.10

`Error: parse error in "minio/templates/statefulset.yaml": template: minio/templates/statefulset.yaml:7: unexpected "=" in operand`

Reply:
It seems like the go version, please refer to https://github.com/helm/helm/issues/4987
BTW, I use Helm 3